control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -1635,6 +1635,1232 @@ def test_service_phase21_readiness_avoids_full_table_list_reads(self) -> None:
+            case_id=promoted_case.case_id,
+            review_owner="analyst-001",
+            intended_outcome="Keep reviewed requests visible on readiness before delegation starts.",
+        )
+        action_request = service.create_reviewed_action_request_from_advisory(
+            record_family="recommendation",
+            record_id=recommendation.recommendation_id,
+            requester_identity="analyst-001",
+            recipient_identity="repo-owner-001",
+            message_intent="Notify the accountable repository owner about the reviewed permission change.",
+            escalation_reason="The approved reviewed request must stay visible until delegation begins.",
+            expires_at=reviewed_at + timedelta(hours=4),
```

Ruff (0.15.10)

[error] Undefined name `timedelta`

(F821)